lib/props/props.test.js

```diff
@@ -0,0 +1,14 @@
+import props from '.'
+
+test('it doesn\'t extract props if element of keys is not an array', () => {
```

Comment:
I think, and it's ramda's behavaior, that `path` should be array, but keys could be strings

Reply:
fixed